src/main/java/org/fundacionjala/coding/abel/HighestAndLowest.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.coding.abel;
+
+
+/**
+ *
+ */
+public class HighestAndLowest {
+
+    /**
+     * @param numbers sdfsd
+     * @return sdfsd
+     */
+
+    public String highAndLow(final String numbers) {
+        String[] container = numbers.split(" ");
+        int mini = Integer.MAX_VALUE;
+        int max = Integer.MIN_VALUE;
+        for (int i = 0; i < container.length; i++) {
+            int num = Integer.parseInt(container[i]);
+            if (num < mini) {
+                mini = num;
+            }
+            if (num > max) {
+                max = num;
+            }
+        }
+
+        return String.join(" ", String.valueOf(max), String.valueOf(mini));
```

Comment:
This may be refactored like this.
```java
    public String highAndLow(final String numbers) {
        int min = Integer.MAX_VALUE;
        int max = Integer.MIN_VALUE;
        String[] container = numbers.split(" ");
        for (String aContainer : container) {
            int num = Integer.parseInt(aContainer);
            if (num < min) {
                min = num;
            } else if (num > max) {
                max = num;
            }
        }
        return String.format("%d %d", max, min);
    }
```